src/test/java/uk/gov/pay/connector/gateway/worldpay/WorldpayPaymentProviderTest.java

```diff
@@ -513,7 +521,7 @@ public void shouldErrorIfWorldpayReturns401() throws Exception {
     }
```

Comment:
On Lines 518-520 I think we could use `assertThrows` and remove the try-catch block with something like this?
```suggestion
var thrown = assertThat(GatewayError.class, () -> providerWithRealGatewayClient.authorise(getCardAuthorisationRequest());
assertThat(thrown.getMessage(), is(...));
    }
```

Reply:
Similarly with any other tests which assert on exceptions.

Reply:
Fixed